src/mpyl/cli/__init__.py

```diff
@@ -1,23 +1,13 @@
 """Command Line Interface parsing for MPyL"""
 
-import asyncio
```

Comment:
Not sure if you already checked, but just in case: can this be removed from Pipfile ?

Reply:
asyncio is part of python